migration/migrating-4_2-4/deploying_cam.adoc

```diff
@@ -0,0 +1,27 @@
+[id='deploying-cam_{context}']
+= Deploying the Cluster Application Migration tool
+include::modules/common-attributes.adoc[]
+:context: migrating-4_2-4_x
+:migrating-4_2-4_x:
+
+toc::[]
+
+Deploying the CAM tool involves the following steps:
```

Comment:
Spell out "CAM" to be consistent throughout. I'll stop leaving redundant comments. :)